helm/svc-bie-kafka/templates/deployment.yaml

```diff
@@ -0,0 +1,35 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: vro-svc-bie-kafka
+  labels: {{- toYaml .Values.labels | nindent 4 }}
+spec:
+  replicas: {{ .Values.replicaCount }}
+  selector:
+    matchLabels: # must match the pod template's labels
+      {{- toYaml .Values.labels | nindent 6 }}
+  template:
+    metadata:
+      labels:
+        sidecar.istio.io/inject: "false"
+        {{- toYaml .Values.labels | nindent 8 }}
+      annotations:
+        {{- include "vro.annotations.pod" . | nindent 8 }}
+        {{- include "vro.annotations.reloadLatest" . | nindent 8 }}
+    spec:
+      {{ include "vro.imagePullSecrets" . | nindent 6 }}
+      containers:
+        - name: svc-bie-kafka{{ include "vro.containerSuffix" . }}
+          image: {{ include "vro.imageRegistryPath" . }}vro-svc-bie-kafka:{{ include "vro.imageTag" . }}
+          env:
+            {{- include "vro.commonEnvVars" . | nindent 12 }}
+            {{- include "vro.mqClient.envVars" . | nindent 12 }}
+            # VRO_SECRETS_BIE_KAFKA provides Kafka credentials (keys and certs)
+            # TODO: Uncomment the following after it is added to Hashicorp Vault and secrets are deployed
+            #- name: VRO_SECRETS_BIE_KAFKA
+            #  valueFrom:
+            #    secretKeyRef:
+            #      name: vro-secrets
+            #      key: VRO_SECRETS_BIE_KAFKA
```

Comment:
Remember to uncomment after it is added to Hashicorp Vault and secrets are deployed. See https://github.com/department-of-veterans-affairs/abd-vro/wiki/Secrets-Vault#addingmodifying-a-secret